customizations/README.md

```diff
@@ -0,0 +1,187 @@
+# Customizations
+
+That file explain how to configure your CARTO selfhosted to meet your needs.
+
+## Production Ready
+
+By default, the Helm configuration with the config files provided by CARTO should work but it's not production ready.
+In the following steps we are going to explain the steps to approximate the installation to a production ready environment.
+
+It should be configured:
+- **MANDATORY** Configure the domain to be used.
+- Allow the external traffic to access the app.
+- Configure DBs to be production ready. Our recomendation is to use managed DBs with backups and so on.
+
+Configuring it is optional:
+- Configure scale of the components (staticaly or with auto-scaling).
+- Use your own bucket to store the data (By default, GCP CARTO buckets are used)
+
+## Architecture diagram
+
+<!--
+We should add an arquitectural diagram to make it easier for customers to understand the parts and the relationship between them.
+-->
+
+## How to define customizations
+
+There are two ways to configure or customize the deployment:
+- [**RECOMMENDED**] Create a dedicated [yaml](https://yaml.org/) file. For example, you can create a file with the next content:
+  ```yaml
+  customConfigValues:
+    selfHostedDomain: "my.domain.com"
+  ```
+  And add the following at the end of ALL the install or upgrade command:
+  ```bash
+  ... -f <my_customization_file>.yaml
+  ```
+- Use the inline set. For example, add the following at the end of ALL the install or upgrade command:
+  ```bash
+  ... --set customConfigValues.selfHostedDomain=my.domain.com
+  ```
+
+
+## Configure the domain of your selfhosted
```

Comment:
should we add here the TLS config? I think it would make sense

Reply:
Mmm. I put it in the section to open to the internet. Remember that the default now it's to not be opened.